test/cookbooks/test/recipes/default.rb

```diff
@@ -14,10 +14,12 @@
 tar_package 'https://www.nano-editor.org/dist/v2.8/nano-2.8.7.tar.gz' do
   prefix '/usr/local'
   creates '/usr/local/bin/nano'
+  checksum '23ffc2de52d687739fed6dc2fc94df36aa7da7bb52c8740c523fdd7336fdbc8c'
 end
 
 tar_extract 'https://www.nano-editor.org/dist/v2.8/nano-2.8.7.tar.gz' do
   target_dir '/usr/local'
   creates '/opt/myapp/mycode/lib'
   tar_flags ['-P', '--strip-components 1']
+  checksum '23ffc2de52d687739fed6dc2fc94df36aa7da7bb52c8740c523fdd7336fdbc8c'
```

Comment:
As this is just exercising the underlying remote_file resource that is used this isn't needed. It's just adding extra work if the package name is updated.